README.md

```diff
@@ -1,3 +1,26 @@
-# S-BLH-roman-numeral-converter
-2nd project of java script
+<<<<<<< HEAD
+# PRoman-Numeral-Converter 🚀
+
+## Project Description 📝
+- 2nd project of js
+- its a converter , that can change latin numbers to roman numbers
+
+## Technologies Used 🛠️
+- HTML
+- CSS
+- JavaScript
+
+## Usage 🎯
+to use this application, you only have to write the greek numbers and then click on the convert, to change them to roman numbers
+
+## Demo 📸
+![Demo]:C:\Users\Black Store\Desktop\Screenshot 2024-06-30 144151.png
```

Comment:
- Here, it is not showing the screenshot, I told you not to use image link from your Disk C or ... instead, add the screenshot to your project folder and then use its link like this `FILE_NAME.FILE_FORMAT` for example `image.png`